src/cluster/sharding.rst

```diff
@@ -179,123 +328,114 @@ looks like this:
         }
     }
 
-After PUTting this document, it's like magic: the shards are now on node2 too!
-We now have ``n=2``!
+Now you can ``PUT`` this new metadata:
 
-If the shards are large, then you can copy them over manually and only have
-CouchDB sync the changes from the last minutes instead.
+.. code:: bash
 
-.. _cluster/sharding/move:
+    $ curl -X PUT $COUCH_URL:5986/_dbs/{name} -d '{...}'
 
-Moving Shards
-=============
+Replicating from old to new
```

Comment:
Nooo no no no. Big -1. Don't do this. We have even talked about disabling the replicator on port 5986 for any purpose other than upgrading from 1.x to 2.x.

Instead just add the shards to the map and CouchDB will manage this thru internal replication itself.